dev-util/ragel/ragel-6.9.recipe

```diff
@@ -29,6 +29,7 @@ REQUIRES="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	cmd:awk
```

Comment:
If awk is needed, then **`cmd:awk`** should be added to BUILD_**PRE**REQUIRES, not BUILD_REQUIRES.